config/default/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+namespace: ds-pipelines-controller
+namePrefix: ds-pipelines-controller-
+resources:
+- ../crd
+- ../rbac
+- ../manager
+
+images:
+  - name: controller
+    newName: quay.io/hukhan/dspipelines-controller
```

Comment:
Is this the right location? :-)

Reply:
no but we don't have it built in the official location yet and that's probably out of scope for this, though it should be a github issue, I  added a todo comment as well